packages/plugin-assets-retry/src/runtime/index.ts

```diff
@@ -99,10 +100,14 @@ function createElement(
     if (attributes.isAsync) {
       script.dataset.rsbuildAsync = '';
     }
-
     return {
       element: script,
-      str: `<script src="${attributes.url}" ${crossOriginAttr} ${retryTimesAttr} ${isAsyncAttr}></script>`,
+      str:
+        // biome-ignore lint/style/useTemplate: template string including html tag causes errors when inlining in html
```

Comment:
can you add some explanations for this

Reply:
done
// use "</" + "script>" instead of script label to avoid syntax error when inlining in html